javaparser-core-testing/src/test/java/com/github/javaparser/utils/TestUtils.java

```diff
@@ -194,13 +194,13 @@ public static void assertExpressionValid(String expression) {
      * Assert that "actual" equals "expected", and that any EOL characters in "actual" are correct for the platform.
      */
     public static void assertEqualsNoEol(String expected, String actual) {
-        assertEquals(normalizeEolInTextBlock(expected, EOL), actual);
+        assertEquals(normalizeEolInTextBlock(expected, EOL), normalizeEolInTextBlock(actual, EOL));
```

Comment:
My understanding of `assertEqualsNoEol` is that it should assert that everything except EOL is equal. Given this, my inclination is that we should normalise to `LineSeparator.ARBITRARY` to explicitly indicate this. 

Note that this would contradict the javadoc which **ALSO** specifies ".. and that any EOL characters in "actual" are correct for the platform.". My preference here would be to have a separate assertion for this.

Reply:
It is there to compare a string with "\n"'s in it to some output which contains some kind of line ending, which is normalized to "\n" so they match. If I remember correctly :-)

Reply:
For the purpose of this assert (and how I've ever used it based on the name), the important bit is that they're the **same** line endings as opposed to being any particular line ending.

The javadoc suggests otherwise though -- that it should match the system's line ending.

Reply:
Ah yes, the other way around. Well, the important thing is that it is really nice to be able to say "\n" in an expectation and not have to bother with Windows/Unix differences. Yeah, I see what you're doing now. Okay.